docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
         environment:
             - POSTGRES_PASSWORD=isamplesinabox
             - POSTGRES_USER=isb_writer
-            - POSTGRES_DB=isb_1
+            - POSTGRES_DB=isb_2
```

Comment:
The issue I ran into here is that the format of the db dump file I used failed miserably if the db already existed.  So, I created a new one and imported into there.  Long term, I think I need to redo the dump to only include data (not table and index creation), and try and import that way.  But short term while figuring this all out I just recreated the db and brought up the container with the new db name but left iSB pointing at the old one (so it wouldn't recreate).  Another option is to change the iSB app to not create the schema.  One of those things that is a still a bit manual at this point.